src/InputContainer.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { ADD_TASK, UPDATE_TASK } from './actions';
+
+import Input from './Input';
+
+function InputContainer() {
+  const taskTitle = useSelector((state) => (state.taskTitle));
+  const dispatch = useDispatch();
+
+  function handleClickAddTask() {
+    dispatch({ type: ADD_TASK });
+  }
+
+  function handleClickTaskTitle(event) {
+    dispatch({ type: UPDATE_TASK, payload: event.target.value });
```

Comment:
```suggestion
    dispatch({
      type: UPDATE_TASK,
      payload: event.target.value
    });
```
인자가 두개 이상일 경우에는 줄바꿈 처리해주는건 어떨까요? 가독성에 도움이 될 것 같아요 ㅎㅎ